source/features/file-finder-buffer.tsx

```diff
@@ -0,0 +1,72 @@
+import features from '../libs/features';
+import select from 'select-dom';
+
+const FILE_FINDER_INPUT_SELECTOR = '.js-tree-finder > .breadcrumb > #tree-finder-field';
+
+let fileFinderBuffer = '';
+
+const isValidCharacter = (char: string): boolean => (
+	char.length === 1
+) && ((
+	char >= 'a' &&
+  char <= 'z'
+) || (
+	char >= 'A' &&
+  char <= 'Z'
+) || (
+	char >= '0' &&
+  char <= '9'
+));
+
+const keyDownHandler = ({key, target}: KeyboardEvent): void => {
+	const nodeName = (target as Element).nodeName;
+	if (nodeName !== 'INPUT' && nodeName !== 'TEXTAREA' && isValidCharacter(key)) {
+		fileFinderBuffer = `${fileFinderBuffer}${key}`;
```

Comment:
This method won’t work as well as an actual input field, for example the backspace and non-latin characters don’t work.

What if you create a hidden `type=text` field (with `visibility:hidden`), focus it and have it receive all the input natively? When the actual FF’s field is available, either copy the value or swap the fields (`ghField.replaceWith(ourField)`, ideal, so it works without interruptions)

The advantage of this is that ****later**** we can make it visible as soon as the user presses `t`. But the styling for this is harder, so keep it hidden for now. I’ll design it next month